class-tapi-query.php

```diff
@@ -59,6 +59,11 @@ private function _resolve_type() {
 
 	public function query() {
 		$this->tweets = WP_Twitter_API()->get( $this->path, $this->query_args );
+		if ( is_object( $this->tweets ) ) {
```

Comment:
consider adding a comment like `// The search endpoint returns an object instead of an array of tweets` or whatever.